Automattic/wp-calypso
# Plugins update manager: Add minutes to 24-hour format options

Related to https://github.com/Automattic/wp-calypso/pull/88595
Related to https://github.com/Automattic/wp-calypso/issues/88417

## Proposed Changes

* Create/edit form: Added minutes to 24-hour format options
* Created common serverSyncCallback object
* Renamed form control components name to follow convention
* Switched badge position for the mobile view matching Figma design

## Testing Instructions



* For an Atomic site, change the site setting to have a 24-hour format
* Go to `/plugins/scheduled-updates`
* Select that Atomic site
* Create or edit an existing schedule
* Check if the format is "01:00"

<img width="706" alt="Screenshot 2024-03-18 at 12 10 30" src="https://github.com/Automattic/wp-calypso/assets/1241413/ca8728e5-df27-434f-8f5c-014469af702d">

| Before | After |
|--------|--------|
| <img width="413" alt="Screenshot 2024-03-18 at 13 41 22" src="https://github.com/Automattic/wp-calypso/assets/1241413/69c3af95-a6f0-47db-89d1-59009949fabe"> | <img width="406" alt="Screenshot 2024-03-18 at 13 41 02" src="https://github.com/Automattic/wp-calypso/assets/1241413/191ed508-3838-4020-bda7-587fd4d014ef"> | 

## Pre-merge Checklist
